app/templates/components/widget-discussions-post.hbs

```diff
@@ -41,23 +41,23 @@
 	</span>
 	<span class="post-counters">
 		<span class="upvote-count">
-			{{svg 'upvote' viewBox='0 0 24 24' class='icon upvote'}}
+			{{svg 'wds-icons-upvote' viewBox='0 0 24 24' class='wds-icon wds-icon-tiny upvote'}}
```

Comment:
why do we use regular size icon and apply wds-icon-tiny class to it? And do we need this viewbox?